### PR TITLE
LPS-151051 Add name required validation in object View and Layout

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-151051

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/index.tsx [hunk 1]
```diff
@@ -206,6 +208,15 @@ const Layout: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 
 		const parentWindow = Liferay.Util.getOpener();
 
+		if (objectLayout.name[defaultLanguageId] === '') {
```
Comment (miltonmc) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/index.tsx:211]: Use `invalidateRequired()` in order to enforce same rules followed in the `InfoScreen.tsx`
```suggestion
		if (invalidateRequired(objectLayout.name[defaultLanguageId])) {
```

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx [hunk 2]
```diff
@@ -159,6 +161,15 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 
 		const parentWindow = Liferay.Util.getOpener();
 
+		if (objectView.name[defaultLanguageId] === '') {
```
Comment (miltonmc) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx:164]: Use `invalidateRequired()` in order to enforce same rules followed in the `BasicInfoScreen.tsx`
```suggestion
		if (invalidateRequired(objectView.name[defaultLanguageId])) {
```

**Review by miltonmc — CHANGES_REQUESTED** [review]
Just some tweaks:

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**